src/components/network/PoweredBy.vue

```diff
@@ -27,17 +28,15 @@
         inactive: !network.powered.providerAddress
       }"
       @click="
-        network.powered.providerAddress && isCurrentNetwork
+        isMenu
```

Comment:
?

Reply:
ah ok, yeah. Maybe it is just better like that. Not having that link in Networks 👍 Yes, good change

Reply:
yeah so now it's only a link when it's in the sidebar.